src/Migration/Sources/Appwrite.php

```diff
@@ -977,15 +979,24 @@ private function exportFiles(int $batchSize)
                 );
 
                 foreach ($response['files'] as $file) {
-                    $this->exportFileData(new File(
-                        $file['$id'],
-                        $bucket,
-                        $file['name'],
-                        $file['signature'],
-                        $file['mimeType'],
-                        $file['$permissions'],
-                        $file['sizeOriginal'],
-                    ));
+                    try {
+                        $this->exportFileData(new File(
+                            $file['$id'],
+                            $bucket,
+                            $file['name'],
+                            $file['signature'],
+                            $file['mimeType'],
+                            $file['$permissions'],
+                            $file['sizeOriginal'],
+                        ));
+                    } catch (\Throwable $e) {
+                        $this->pushError(new Exception(
```

Comment:
Name is correct, but we would usually use a name like `addError` in utopia

Reply:
Alright, will alter accordingly